app/webpacker/styles/trainee_list.scss

```diff
@@ -32,7 +32,7 @@
       display: flex;
       justify-content: space-between;
 
-      .app-export--link {
+      .app-export-link-container {
```

Comment:
renamed this to link-container, as we use this styling now for the Admin text too (not a link). It's not used strictly for links hence the renaming.